Add Samples Index metadata for samples

tjohns opened the pull request:
Add .google/packaging.yaml file to each sample directory, so that the
samples can be picked up by Google's Samples Index system.

Also updated README.md for hello-kotlin sample to add "Kotlin" to the
sample's title.

tjohns committed:
Add Samples Index metadata for samples

Add .google/packaging.yaml file to each sample directory, so that the
samples can be picked up by Google's Samples Index system.

Also updated README.md for hello-kotlin sample to add "Kotlin" to the
sample's title.

keyboardsurfer commented on configuration-apks/.google/packaging.yaml:
```diff
@@ -0,0 +1,41 @@
+# See http://go/sample-categories
+technologies: [Android]
+categories:   [Instant Apps]
+languages:    [Java]
```
Kotlin

tjohns replied:
Done

keyboardsurfer commented on cookie-api/.google/packaging.yaml:
```diff
@@ -0,0 +1,41 @@
+# See http://go/sample-categories
+technologies: [Android]
+categories:   [Instant Apps]
+languages:    [Java]
```
Kotlin

tjohns replied:
Done

keyboardsurfer commented on install-api/.google/packaging.yaml:
```diff
@@ -0,0 +1,41 @@
+# See http://go/sample-categories
+technologies: [Android]
+categories:   [Instant Apps]
+languages:    [Java]
```
Kotlin

tjohns replied:
Done

keyboardsurfer commented on storage-api/.google/packaging.yaml:
```diff
@@ -0,0 +1,41 @@
+# See http://go/sample-categories
+technologies: [Android]
+categories:   [Instant Apps]
+languages:    [Java]
```
Kotlin

tjohns replied:
Done

keyboardsurfer reviewed (commented):
Thanks for the PR.
I added some comments, where the language differs from your proposed change. Please update and this PR is good to go.

tjohns committed:
Update metadata for 4 Kotlin samples

Marked the following samples as using the Kotlin language:

- cookie-api
- storage-api
- install-api
- configuration-apks

tjohns reviewed (commented):
Changes submitted. PTAL.
